src/mame/drivers/hp9k_3xx.cpp

```diff
@@ -296,6 +303,15 @@ void hp9k3xx_state::machine_reset()
 {
 }
 
+WRITE8_MEMBER(hp9k3xx_state::gpib_w)
+{
+}
```

Comment:
Could you please put a configurable log message here, using the stuff from `logmacro.h`?  I'd rather not see unknown stuff blackhole.

Reply:
done